packages/tools/src/utilities/segmentation/createBidirectionalToolData.ts

```diff
@@ -38,9 +38,7 @@ export default function createBidirectionalToolData(
     invalidated: true,
     metadata: {
       toolName: 'Bidirectional',
-      viewPlaneNormal,
-      viewUp,
-      FrameOfReferenceUID,
+      ...viewport.getViewReference({ sliceIndex }),
```

Comment:
why {sliceIndex}? what is the different between viewport.getViewReference() and viewport.getViewReference({sliceIndex}) outputs?